manage/calendars.rst

```diff
@@ -1,7 +1,7 @@
 Calendars
 *********
 
-A calendar is required to calculate escalations or reports based on business hours.
+A calendar is required to calculate escalations, reports based on business hours or for trigger execution.
```

Comment:
This new version of the sentence is a little messy. The intended structure is:

> **Before:** A calendar is required to calculate (escalations OR reports)  
> **After:** A calendar is required (to calculate (escalations OR reports) OR for trigger execution)

### One solution

Clarify the structure with the right punctuation and conjunctions ("and"/"but"/"or"-type words):

> A calendar is required to calculate escalations or reports based on business hours, or for trigger execution.

In other words, "A calendar is required (to calculate (escalations `or` reports) `, or` for trigger execution)."

### A better solution

The suggestion above is not ideal because some parts of the sentence are wordy (like "reports based on business hours"), and that makes it hard to see the overall structure.

A better way would be to simplify the structure of the sentence so that it's easier to see at a glance:

```suggestion
A calendar is required to:

* automate ticket escalations,
* generate reports that only capture activity during business hours, or
* set up time-sensitive triggers.
```

(And just to make sure I understand you: are calendars required to execute _any_ triggers? or only to execute the new time-sensitive triggers introduced in v3.3?)

Reply:
Thank you very much - you're right, that sentence really is messy.
The listing is a lot better!